File: Java/question4.java
import java.util.*;
class question4{
  public static void main(String args[]){
    Scanner input = new Scanner(System.in); // 輸入工具
    int times = Integer.parseInt(input.nextLine());
    for(int i = 0 ; i < times ; i++){
    String data = input.nextLine();
    String[] numbers = data.split(" ");
    int total = 0;
    for(String x : numbers){
      total = total + Integer.parseInt(x);
    }
    float answer = total/(float)numbers.length;
    System.out.printf("%.3f\n",answer);
  }
  }
}

File: Python/question4.py
"""
 4. 測試範例 平均(2)
"""
times = int(input()) # 輸入次數

for i in range(times):
  myInput = input().split(' ') # 切割字串
  answer = 0
  for j in myInput:
    answer = answer + int(j)
  print("{:.3f}".format(answer/len(myInput))) # ”{:.3f}“ --> 小數點第三位 len()用來計算陣列大小